File: c++/operators/operators_arithmetic.cpp
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */
// SPDX-License-Identifier: GPL-3.0-or-later

#include <iostream>

int main()
{
  int a = 6, b = 2, c;

  c = a + b;
  std::cout << a << " + " << b << " = " << c << std::endl;
  c = a - b;
  std::cout << a << " - " << b << " = " << c << std::endl;
  c = a * b;
  std::cout << a << " * " << b << " = " << c << std::endl;
  c = a / b;
  std::cout << a << " / " << b << " = " << c << std::endl;
  c = a % b;
  std::cout << a << " % " << b << " = " << c << std::endl;

  std::cout << "-" << a << " = " << -a << std::endl;
  std::cout << "+" << b << " = " << +b << std::endl;
}

File: perl/operators/operators_arithmetic.pl
#
# Copyright (C) 2019-2026 Katarzyna Krasińska
# Languagium - https://github.com/katheroine/languagium
# Licensed under GPL-3.0 - see LICENSE.md
#

use strict;
use warnings;

my $a = 6; my $b = 2; my $c;

$c = $a + $b;
print("$a + $b = $c\n");
$c = $a - $b;
print("$a - $b = $c\n");
$c = $a * $b;
print("$a * $b = $c\n");
$c = $a / $b;
print("$a / $b = $c\n");
$c = $a % $b;
print("$a % $b = $c\n");
$c = $a ** $b;
print("$a ** $b = $c\n");

print("-$a = " . -$a . "\n");
print("+$b = " . +$b . "\n");
